AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSDBDocumentStore.m

```diff
@@ -77,13 +77,16 @@ - (BOOL)upsertWithToken:(MSTokenResult *)token
   // This is the same as [[NSDate date] timeIntervalSince1970] - but saves us from allocating an NSDate.
   NSTimeInterval now = NSDate.timeIntervalSinceReferenceDate + NSTimeIntervalSince1970;
   NSString *tableName = [MSDBDocumentStore tableNameForPartition:token.partition];
+
+  // If operation is nil, pass NULL value, else user the operation name in this format '<OPERATION_NAME>' (note the single quotes).
+  NSString *normalizedOperationString = operation != nil ? [NSString stringWithFormat:@"'%@'", operation] : nil;
```

Comment:
Does passing `nil` here actually works? Or should you pass the string `"NULL"`? I am doing the later one somewhere else, but `nil` might just work.

Reply:
tried it, it's working

Reply:
I would still use "NULL" for consistency.

Reply:
That's what we use in core too.